.azure/scripts/install_cosign.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+ARCH=$1
+if [ -z "$ARCH" ]; then
+    ARCH="amd64"
+fi
+
+curl -L https://github.com/sigstore/cosign/releases/download/v2.2.0/cosign-linux-${ARCH} > cosign && chmod +x cosign
```

Comment:
what about using a variable to define the cosign version as you do in the script for installing syft.
It's easier to identify what to update when we want to move to a newer cosign version.